cirq/circuits/qasm_output.py

```diff
@@ -124,7 +124,7 @@ def __repr__(self) -> str:
 
 
 class QasmOutput:
-    valid_id_re = re.compile('[a-z][a-zA-Z0-9_]*\Z')
+    valid_id_re = re.compile('[a-z][a-zA-Z0-9_]*\Z') # pylint: disable=anomalous-backslash-in-string
```

Comment:
This warning is correct! Don't disable it, switch to an r-string `r'[a-z][a-zA-Z0-9_]*\Z'`.